public/static/partials/user_host_terminate.html

```diff
@@ -3,8 +3,14 @@
     <span class="semi-muted">
       Terminate <strong>[[curHostData.distro._id]]</strong> host <strong>[[concatName(curHostData.id, curHostData.display_name)]]</strong>?
 	</span>
+    <div ng-show="curHostData.no_expiration" style="margin-bottom:5px">
+      <span class="semi-muted">
+        Host is unexpirable. If you'd really like to terminate, please type "delete":
+      </span>
+      <input type="text" id="input-type"  ng-model="checkDelete"/>
```

Comment:
If the user starts typing it for one host and switches to another, or starts typing it and closes and reopens the modal it looks like what they started typing will already be populated.
Should we clear `checkDelete` when the modal is closed?